README.md

```diff
@@ -161,72 +157,44 @@ You can access a clickable demo with many examples and a [live code editor](http
+
+### Plugins
+You can extend react-carousel default behavior by applying plugins shipped within carousel or by creating and applying your own ones
+
+[**Plugins documentation**](https://beghp.github.io/gh-pages-rc-15/docs/api/plugins)
 
 ### Dots props
 
 | Prop | Type | Default | Description |
 | --- | --- | --- | --- |
-| [**number**](https://brainhubeu.github.io/react-carousel/docs/examples/dots) | *Number* | Amount of slides | Number of slides in the carousel you want to control |
+| [**value**](https://brainhubeu.github.io/react-carousel/docs/examples/dots) | *Number* | slide position in the slides Array | Current `Carousel` value |
 | [**onChange**](https://brainhubeu.github.io/react-carousel/docs/examples/dots) | *Function* | `undefined` | `onChange` callback (works the same way as `onChange` in `Carousel` component) |
-| **rtl** | *Boolean* | `false` | Indicating if the dots should have direction from Right to Left |
+| [**number**](https://brainhubeu.github.io/react-carousel/docs/examples/dots) | *Number* | Amount of slides | Number of slides in the carousel you want to control |
 | [**thumbnails**](https://brainhubeu.github.io/react-carousel/docs/examples/thumbnails) | *Array of ReactElements* | `undefined` | Array of thumbnails to show. If not provided, default dots will be shown |
-| [**value**](https://brainhubeu.github.io/react-carousel/docs/examples/dots) | *Number* | slide position in the slides Array | Current `Carousel` value |
-
-## Contributing
-[The GitHub issues list](https://github.com/brainhubeu/react-carousel/issues) is our roadmap.
-You're more than welcome to vote:
-- with 👍if you like a given feature request or you'd like a given bug to be fixed
-- with ❤️ if you love a given feature request or fixing a given bug is critical for you
-- with 👎if in your opinion, a given feature would create more damages than the value provided by it or you consider a given bug to be a feature
-
-We don't give any guarantee to fix even the most liked issues but 👍and ❤️ increase probability of fixing while 👎decreases the probability of fixing.
-
-You're also more than welcome to:
-- submit a feature request
-- report a bug
-- ask a question
-- comment an issue, discussing the details
-- open a PR, fixing a given issue
+| [**rtl**](https://brainhubeu.github.io/react-carousel/docs/examples/rtl) | *Boolean* | `false` | Indicating if the dots should have direction from Right to Left |
 
 ### Setting up local development which means running the docs/demo locally:
 - `git clone https://github.com/brainhubeu/react-carousel`
 - `cd react-carousel`
 - `yarn`
-- `cd docs-www`
-- `yarn`
-- if you want to connect demo with the carousel source code, replace `__RC_ENV__` into `development` in https://github.com/brainhubeu/react-carousel/blob/master/docs-www/src/globalReferences.js#L2 and remove the `.babelrc` file in the root directory; otherwise, it will use the carousel code installed in `docs-www/node_modules`
```

Comment:
does `yarn start-demo` use the code from `src` instead from `node_modules/@brainhubeu/react-carousel`?

Reply:
yes, it uses code from src